Source/WebKit/WebProcess/WebPage/Cocoa/UnifiedTextReplacementController.mm

```diff
@@ -363,8 +392,47 @@
         ASSERT_NOT_REACHED();
         return;
     }
+    auto replacedRange = resolvedRange;
+
+    auto subCharacterRange = [](WebCore::CharacterRange superRange, WebCore::CharacterRange previousRange) {
+        auto location = previousRange.location + previousRange.length;
+        return WebCore::CharacterRange(location, superRange.length - previousRange.length);
+    };
+
+    for (auto session : m_textIndicatorCharacterRangesForSessions) {
+        if (session.first == uuid) {
```

Comment:
Can we bail from iteration once we find the one we're looking for? (`break` at the end) Would this read better with an early-continue in the not-the-right-UUID case?